pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -39,28 +43,37 @@ const (
 	// Name is the name of the plugin used in the plugin registry and configurations.
 	Name = "NodeAffinity"
 
-	// ErrReason for node affinity/selector not matching.
-	ErrReason = "node(s) didn't match node selector"
+	// ErrReasonPod is the reason for Pod's node affinity/selector not matching.
+	ErrReasonPod = "node(s) didn't match Pod's node affinity"
+
+	// errReasonDefault is the reason for default node affinity not matching.
+	errReasonDefault = "node(s) didn't match scheduler's default node affinity"
```

Comment:
tweak the message to drop "default", maybe something like `"node(s) didn't match scheduler-enforced node affinity"`

Reply:
Done